src/movies.js

```diff
@@ -1,22 +1,77 @@
 // The `movies` array from the file `src/data.js`.
-console.log('movies: ', movies);
+// console.log('movies:', movies);
 
+// const movies = require('./data');
 
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
+// const arrayDirector = [];
+function getAllDirectors(arrOfMovies) {
+  return (arrayDirector = arrOfMovies.map(function (element, value) {
+    // console.log(element);
+    return element.director;
+    // console.log(element);
+  }));
```

Comment:
I like that you are returning result straight away! same as in below iterations :)